### PR TITLE
[documentation] Explain implicit profiles

### PR DESCRIPTION
**Pull request by andrewfg**
This PR adds an explanation of how the framework uses implicit profiles for items where no profile is explicitly defined.

See https://github.com/openhab/openhab-addons/pull/15656#issuecomment-1763059699

Signed-off-by: Andrew Fiddian-Green <software@whitebear.ch>

### CODE REVIEW

**Comment by andrewfg** [comment]
@lolodomo for info..

File: configuration/items.md [hunk 1]
```diff
@@ -731,6 +731,23 @@ If this is the case, you will find those within the documentation of the Binding
...
 | `transform:<SERVICE>`                                                                         | State   | All                   | Transformation Profiles can be used to transform the item state (instead of only transforming it in the [Sitemap]({{base}}/ui/sitemaps.html)) using the specified transformation service. You can find the documentation of these Profiles within the [documentation of the Transformation Service](/addons/#transform). E.g. [map](addons/transformations/map/#usage-as-a-profile) or [jsonpath](addons/transformations/jsonpath/#usage-as-a-profile)                                        |
 
+##### Implicit Profiles
+
+If an Item does not have an explicitly defined Profile the framework actually uses 'behind the scenes' an implicit profile.
```
Comment (stefan-hoehn) [inline configuration/items.md:736]: uses an implicit profile 'behind the scenes'.

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]
Very good addition to the docs here. See my small recommendation.

File: configuration/items.md (hunk 1 above)
Comment (andrewfg) [reply configuration/items.md:736]: Fixed

**Comment by lolodomo** [comment]
There is another page where profiles are defined and this page is not clear about defaults.
I have to find this page...

**Comment by stefan-hoehn** [comment]
@lolodomo Should I wait to merge until your request has been done by @andrewfg ?

**Comment by lolodomo** [comment]
> @lolodomo Should I wait to merge until your request has been done by @andrewfg ?

In case @andrewfg can handle it before OH 4.1 is released, yes please.

**Comment by andrewfg** [comment]
> There is another page where profiles are defined and this page is not clear about defaults.

@lolodomo my new '**Implicit Profiles**' chapter is a direct sub chapter of the '**Profiles**' chapter so I am not clear on what extra you want me to add?

**Comment by lolodomo** [comment]
> @lolodomo my new '**Implicit Profiles**' chapter is a direct sub chapter of the '**Profiles**' chapter

Sorry, I did not realize that.
So it should be fine.
Please ignore my previous comment.

**Review by stefan-hoehn — APPROVED** [review]
Thanks, Andrew, for the contribution. 
Looks good to me as well.

**Pull request merged**